lib/glific/flows/translate/google_translate.ex

```diff
@@ -3,11 +3,27 @@ defmodule Glific.Flows.Translate.GoogleTranslate do
   Code to translate using google translate as the translation engine.
   """
   @behaviour Glific.Flows.Translate.Translate
-  @google_translate_params %{"temperature" => 0, "max_tokens" => 12_000}
 
   alias Glific.Flows.Translate.Translate
   require Logger
 
+  @language_codes %{
+    "English" => "en",
+    "Hindi" => "hi",
+    "Tamil" => "ta",
+    "Kannada" => "kn",
+    "Malayalam" => "ml",
+    "Telugu" => "te",
+    "Odia" => "or",
+    "Assamese" => "as",
+    "Gujarati" => "gu",
+    "Bengali" => "bn",
+    ~c"Punjabi" => "pa",
+    "Marathi" => "mr",
+    "Urdu" => "ur",
+    "Spanish" => "es"
+  }
+
```

Comment:
instead of hard coding this create a function in `lib/glific/settings/language.ex` which fetches all languages and returns map in same format, making this scalable